### PR TITLE
OCPBUGS-17699: Update WPC GM config

### PR DESCRIPTION
**Pull request by josephdrichard**
Pins should be configured in their entirety

Add required changes to ubxtool commands

### CODE REVIEW

**Review by imiller0 — APPROVED** [review]
/lgtm

**Review by imiller0 — APPROVED** [review]
/lgtm

File: ztp/source-crs/PtpConfigGmWpc.yaml [hunk 1]
```diff
@@ -19,50 +19,63 @@ spec:
     plugins:
       e810:
         enableDefaultConfig: false
-        pins:
-          "$iface_master":
-            "U.FL2": "0 2"
-            "U.FL1": "0 1"
-            "SMA2": "0 2"
-            "SMA1": "0 1"
+        pins: $e810_pins
```
Comment (yliu127) [inline ztp/source-crs/PtpConfigGmWpc.yaml:22]: What does user need to specify here?

**Pull request merged**